File: Exercicios/lista05/q4.cpp
#include <iostream>
using namespace std;

int Aprovado(int note1, int note2){
    int media = (note1 * 2 + note2 * 3) / 5;

    if (media >= 60)
        cout << "O Aluno está Aprovado? True" << endl;
    else
        cout << "O Aluno está Aprovado? False" << endl;
    
    return 0;
}

int main(){
    int note1, note2;

    cout << "Informe a Primeira Nota: ";
    cin >> note1;
    cout << "Informe a Segunda Nota: ";
    cin >> note2;

    Aprovado(note1, note2);
    return 0;
}

File: Exercicios/lista05_Python_C++/q4.py
n1, n2 = map(int, input("Informe as Duas Notas: ").split())

def aprovado(n1, n2):
    media = (n1 * 2 + n2 * 3) // 5
    if media >= 60:
        return True
    else:
        return False

print(f"O Aluno está Aprovado? {aprovado(n1, n2)}")